source/settings/command-line-options.txt

```diff
@@ -75,6 +70,18 @@ configuration file.
        Specify a passphrase to decrypt the :ref:`imported favorite
        connections <compass-import-connections-cli-encrypted>` file. 
 
+   * - .. option:: --showExampleConfig
+     - Provides an example |compass-short| configuration file. 
+
+   * - .. option:: --trustedConnectionString
+     - Suppresses warnings about disallowed connection string properties and allows 
+       automatic connection. Compass inspects the connection string passed in the 
+       command line, and verifies whether it contains parameters that you may 
+       not want to have set and can result in a security risk. 
+       
+       Only set this option if a known trusted connection string is being passed to 
+       |compass-short|.
```

Comment:
[suggestion] we could probably make this active voice

```suggestion
       Only set this option if you pass a known trusted connection string 
       |compass-short|.
```

Reply:
Updated!